config/database.js

```diff
@@ -5,5 +5,5 @@
  * Note that `mongo` name should be same as mongoDB service name defined in docker-compose.yml
  */
 module.exports = {
-    'url' : 'mongodb://mongo:27017'
+    'url' : 'mongodb://localhost:27017/open-library'
```

Comment:
localhost won't work, Please do read the comment. We should use docker service name as the host, rename it to mongo